planning/behavior_velocity_planner/src/scene_module/virtual_traffic_light/scene.cpp

```diff
@@ -19,9 +19,20 @@
 #include <tier4_v2x_msgs/msg/key_value.hpp>
 
 #include <algorithm>
+#include <limits>
 #include <string>
 #include <vector>
 
+namespace tier4_autoware_utils
+{
+template <>
+inline geometry_msgs::msg::Pose getPose(
+  const autoware_auto_planning_msgs::msg::PathPointWithLaneId & p)
```

Comment:
Oh, is this in `autoware_auto_planning_msgs`, not `tier4_planning_msgs`?

Reply:
https://github.com/autowarefoundation/autoware.universe/issues/315#issuecomment-1023946428